app/src/main/java/com/hedvig/app/feature/offer/ui/OfferActivity.kt

```diff
@@ -158,6 +159,13 @@ class OfferActivity : BaseActivity(R.layout.activity_offer) {
                             documentAdapter.submitList(viewState.documents)
                             bottomOfferAdapter.submitList(viewState.bottomOfferItems)
                             setSignState(viewState.signMethod)
+
+                            TransitionManager.beginDelayedTransition(binding.offerToolbar)
```

Comment:
Man, wouldn't it be cool to have an extension for this? Like:

```kotlin
binding.offerToolbar.beginDelayedTransition()
```

Not a necessity, but it could be a niceity 😬